apps/wallet-mobile/src/components/AmountItem/AmountItem.tsx

```diff
@@ -18,9 +20,18 @@ export type AmountItemProps = {
   status?: string
   inWallet?: boolean
   variant?: 'swap'
+  priceImpactRisk?: SwapPriceImpactStatus
```

Comment:
hey @SorinC6 I think your denomination is more precise let's rename `SwapPriceImpactStatus` to `SwapPriceImpactRisk` -> `'none' | 'high' | 'moderate'`